src/sh/emojicoin/wait-for-localnet.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+check_endpoint() {
+	curl -s -f -o /dev/null http://localhost:8070/
```

Comment:
See notes about checking with `jq`.

Reply:
Discussed on call - check that the `not_ready` length of the healthcheck endpoint with `jq` has a length of 0:

Something like:

```shell
# Credit to Bogdan :p
jq '.notReady | length'
```

Reply:
Fixed and added to the base image in `cloud-infra`: https://github.com/econia-labs/cloud-infra/pull/11